src/nvim/change.c

```diff
@@ -1203,7 +1203,7 @@ int open_line(int dir, int flags, int second_line_indent, bool *did_do_comment)
   end_comment_pending = NUL;
   if (flags & OPENLINE_DO_COM && dir == FORWARD) {
```

Comment:
I made a typo here.

Reply:
I'll add a test for that: <https://github.com/vim/vim/pull/9718>, and fix the typo when porting the patch.